jre8.txt

```diff
@@ -0,0 +1,41 @@
+[Section]
+Name = Java SE Runtime Environment 8
+Version = 8u11
+License = Oracle Binary Code License
+Description = Platform and libraries needed for running Java-based programs.
+Size = 29.7 MiB
+Category = 14
+URLDownload = http://svn.reactos.org/packages/jre-8u11-windows-i586.exe
```

Comment:
Java has an archive of these releases here: http://www.oracle.com/technetwork/java/javase/downloads/java-archive-javase8-2177648.html

Reply:
An Oracle account is needed to access these links.